src/core/agent/agent.ts

```
@@ -328,6 +333,63 @@ class Agent {
     });
   }
 
+  async devPreload() {
+    try {
+      await SecureStorage.get(KeyStoreKeys.APP_PASSCODE);
+    } catch (error) {
```

We need to re-throw the error if it's a different error